### PR TITLE
Update text to clarify data sources

### PR DESCRIPTION
**Pull request by tdivoll**
Let's try this again! I was able to rebase, but still lost the changes. I added them back in on the clean branch and this looks better for the PR. Please check to make sure I represented the agencies appropriately for each page.

### CODE REVIEW

File: src/views/datasets/osom/summary.vue [hunk 1]
```diff
@@ -1,12 +1,14 @@
 <template>
   <BuoyDashboard :scenarios="scenarios">
     <template #available-data>
-      The Ocean State Ocean Model (OSOM) is an application of the Regional Ocean
-      Modeling System spanning the Rhode Island waterways, including
-      Narragansett Bay, Mt. Hope Bay, larger rivers, and the Block Island Shelf
-      circulation from Long Island to Nantucket. Data is available by the year
-      at 1.5 hour increments. As the model covers the entire Narragansett Bay,
-      data is always available at all buoy locations.
+    The Ocean State Ocean Model (OSOM), developed in a collaboration between
```
Comment (mcmcgrath13) [inline src/views/datasets/osom/summary.vue:4]: could double check you ran `npm run lint`? A bit of the indenting/line breaks looks a bit off to me (but not very off, so it also might be right 🤷)

File: src/views/datasets/osom/summary.vue [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
+    the University of Rhode Island and Brown University, is an application
+    of the Regional Ocean Modeling System (ROMS) spanning Rhode spanning Rhode
```
Comment (mcmcgrath13) [inline src/views/datasets/osom/summary.vue:6]:
```suggestion
    of the Regional Ocean Modeling System (ROMS) spanning Rhode
```

File: src/views/datasets/osom/summary.vue [hunk 3]
```diff
@@ -1,12 +1,14 @@
...
+    Island waterways, including Narragansett Bay, Mt. Hope Bay, larger rivers,
+    and the Block Island Shelf circulation from Long Island to Nantucket.
```
Comment (mcmcgrath13) [inline src/views/datasets/osom/summary.vue:8]: this sentence is pretty long - maybe break it up?

File: src/views/datasets/plankton/summary.vue [hunk 4]
```diff
@@ -1,8 +1,9 @@
 <template>
   <BuoyDashboard :scenarios="scenarios">
     <template #available-data>
-      Narragansett Bay Long-Term Plankton Time Series is one of the world’s
-      longest-running plankton surveys. Beginning in 1957, weekly samples have
+      Narragansett Bay Long-Term Plankton Time Series, upheld by the University of
```
Comment (mcmcgrath13) [inline src/views/datasets/plankton/summary.vue:4]: is `upheld` the right word here?

File: src/views/datasets/ri-buoy/summary.vue [hunk 5]
```diff
@@ -2,7 +2,10 @@
   <BuoyDashboard :scenarios="scenarios">
     <template #available-data>
       This dataset spans from {{ store.minDate.slice(0, 4) }} to
-      {{ store.maxDate.slice(0, 4) }}. The heatmap below summarizes the number
+      {{ store.maxDate.slice(0, 4) }}. and was collected by the Narragansett
```
Comment (mcmcgrath13) [inline src/views/datasets/ri-buoy/summary.vue:5]:
```suggestion
      {{ store.maxDate.slice(0, 4) }} and was collected by the Narragansett
```

File: src/views/datasets/ma-buoy/summary.vue [hunk 6]
```diff
@@ -3,7 +3,10 @@
     <template #available-data>
       <p>
         This dataset spans from {{ store.minDate.slice(0, 4) }} to
-        {{ store.maxDate.slice(0, 4) }}. The heatmap below summarizes the number
+        {{ store.maxDate.slice(0, 4) }} and was collected by the Narragansett
+        Bay Fixed Station Monitoring Network (NBFSMN) with MassDEP as the lead agency. 
+        Agencies involved in collection and maintenance of the data are: RIDEM-OWR, 
+        URI/GSO MERL, SNEP, and MassDEP. The heatmap below summarizes the number
```
[the remark this answers is not in the document]
Comment (tdivoll) [reply src/views/datasets/ma-buoy/summary.vue:9]: I pulled these from the MassDEP site, deleted this statement. RIDEM will let us know if we need to add any back in.

[the remark this answers is not in the document]
Comment (mcmcgrath13) [reply src/views/datasets/ma-buoy/summary.vue:9]: Yeah, I'm a bit more hesitant to touch the MA page as MassDEP is the data owner there and I think we'd need them to sign off on changes there

File: src/components/base/SideBar.vue [hunk 7]
```diff
@@ -104,12 +104,12 @@ const collapsed = ref(false);
...
 const DATASETS = [
   {
-    name: "RI Buoy Data",
+    name: "RI Buoys (NBFSMN)",
     route: "ri-buoy",
     available: true,
   },
   {
-    name: "MA Buoy Data",
+    name: "MA Buoy (MassDEP/NBFSMN)",
```
Comment (mcmcgrath13) [inline src/components/base/SideBar.vue:112]:
```suggestion
    name: "MA Buoys (MassDEP/NBFSMN)",
```

File: src/views/datasets/osom/summary.vue [hunk 8]
```diff
@@ -1,12 +1,13 @@
...
-      The Ocean State Ocean Model (OSOM) is an application of the Regional Ocean
-      Modeling System spanning the Rhode Island waterways, including
-      Narragansett Bay, Mt. Hope Bay, larger rivers, and the Block Island Shelf
-      circulation from Long Island to Nantucket. Data is available by the year
-      at 1.5 hour increments. As the model covers the entire Narragansett Bay,
-      data is always available at all buoy locations.
+      The Ocean State Ocean Model (OSOM), developed in a collaboration between
+      the University of Rhode Island and Brown University, is an application of the
+      Regional Ocean Modeling System (ROMS). Spatial coverage of the model 
+      spans Rhode Island's major waterways: Narragansett Bay, Mt. Hope Bay, larger rivers,
```
Comment (mcmcgrath13) [inline src/views/datasets/osom/summary.vue:7]:
```suggestion
      Regional Ocean Modeling System (ROMS). The model 
      spans Rhode Island's major waterways: Narragansett Bay, Mt. Hope Bay, larger rivers,
```

File: src/views/datasets/plankton/summary.vue [hunk 9]
```diff
@@ -2,7 +2,8 @@
   <BuoyDashboard :scenarios="scenarios">
     <template #available-data>
       Narragansett Bay Long-Term Plankton Time Series is one of the world’s
-      longest-running plankton surveys. Beginning in 1957, weekly samples have
+      longest-running plankton surveys maintained by the University of
+      Rhode Island. Since 1957, weekly samples have
```
Comment (mcmcgrath13) [inline src/views/datasets/plankton/summary.vue:6]: I can suggest for silly github reasons, but maybe `The Narragansett Bay Long-Term Plankton Time Series, mainted by the University of Rhode Island, is one of the world's longest-running plankton surveys.`

The current phrasing makes it sounds like it's the longest running survey at uri as opposed to the world

**Review by mcmcgrath13 — APPROVED** [review]
Looks good to me! Could you send the preview link to Jeff and Heather to review/approve before we merge?

**Pull request merged**